### DISCUSSION TITLE
tab loading spinner

### DISCUSSION THREAD

**Discussion by N3C2L** [madprops/grasshopper#63] [Ideas]
I like to have a visual feedback when a page loads and also when it has finished loading.The spinner on the favicon could spin while the website is loading and should go away again when the page is finished loading.

**Comment by madprops** [comment]
Not sure this information is available to the tab object. Plus, there's already the Unread icon on newly opened tabs which gives a similar kind of feedback.

**Comment by N3C2L** [comment]
In the "Tree Style tabs" addon a red hourglass replaces the favicon if you load an unloaded tab (or if you open a new tab in the foreground or background). This is useful as a loading indicator, and I would love this addition in Grasshopper. Maybe you can find the solution here: https://github.com/piroor/treestyletab

**Comment by madprops** [comment]
Ok I think this might be possible. The tabs have a `status` property. I'll check if that's updated.

**Comment by madprops** [comment]
I added this but right now I'm using an opacity animation to make it blink a bit instead of using a spinner.

**Comment by N3C2L** [comment]
Very nice, thanks for implementing it.

**Comment by madprops** [comment]
This is uploaded now. For now there's 2 effects available, I might add more later.

**Comment by N3C2L** [comment]
I am enjoying both of them, thanks.

**Reply by madprops** [reply]
A while ago I added an icon effect too.

**Reply by N3C2L** [reply]
Really great, thank you very much.

**Comment by N3C2L** [comment]
This idea has been implemented, so I can close this thread.
